src/types/brand.ts

```diff
@@ -1 +1,4 @@
-export type Brand = 'as24' | 'ms24';
+export enum Brand {
+  AutoScout24 = 'autoscout24',
+  MotoScout24 = 'motoscout24',
```

Comment:
I'am just wondering if we want to be in sync with what we have in listings project. In listings project we have `Brand.Ms24` `Brand.As24`.

Did you take that into consideration?

Reply:
Yea yea, this will be applied to all projects, listing, seller...Idea is to align everything with BE